src/com/belsoft/project/CustomDictionary.java

```diff
@@ -0,0 +1,64 @@
+package com.belsoft.project;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomDictionary {
+
+    private final Map<String, String> names = new HashMap<>();
```

Comment:
We intend to store user name (key) and user information (value), so I would expect to have a way to model a User (one solution: create a User class, with relevant fields)